e2e/ui/global-setup.js

```diff
@@ -1,6 +1,6 @@
 import fs from "fs";
 import { chromium } from "@playwright/test";
-import { client, NOMAD_ADDR, NOMAD_TOKEN } from "./api-client.js";
+import { client, NOMAD_PROXY_ADDR, NOMAD_ADDR, NOMAD_TOKEN } from "./api-client.js";
```

Comment:
`nit`: But we should be able to remove `NOMAD_ADDR`  from this list of imports as its unused now, right?

Reply:
I think so, assuming that `client` is already configured for the HTTP API at that point and doesn't need to be handed addresses?

Reply:
Ok, great! That sounds right to me!